examples/create_machine_pool/README.md

```diff
@@ -0,0 +1,69 @@
+# Creating a Machine Pool for existing Openshift cluster
+
+This Terraform example creates a Node Pool for existing OCM cluster. Creating new Machine pool allows users to add different machine types in the cluster..
+
+## Prerequisites
+
+* You have an AWS account.
+* You have installed the latest version ROSA CLI (`rosa`).
+* You must have an offline OCM token. This token can be generated in the [Red Hat Hybrid Cloud Console](https://console.redhat.com/openshift/token).
+* You have installed Terraform. See the [Hashicorp Terraform page](https://developer.hashicorp.com/terraform/downloads) for the latest version.
+* You have already created OCM cluster.
+
+
+## ROSA cluster creation
+
+1. To run the `terraform apply` you need to set up some variables. This guide uses environmental variables. For more on Terraform variables, see [Managing Variables](https://developer.hashicorp.com/terraform/enterprise/workspaces/variables/managing-variables) in the Terraform documentation.
+
+   > **NOTE**: If you exported these variables in your current command-line session when running the account-roles Terraform example, you do not need to export them again.
+
+   Run the following commands to export your variables. Provide your values in lieu of the brackets. Note that any values declared in the `variables.tf` are the default values if you do not export a superseding value.
```

Comment:
Do you want to mention here other possible variables? or link the [terraform.io](https://registry.terraform.io/providers/terraform-redhat/ocm/latest/docs/resources/machine_pool) page?